src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -3048,6 +3051,22 @@ private void ToggleProcessList()
         processListWindow.Visible = !processListWindow.Visible;
     }
 
+    private void OnMicheViewRequested()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        if (Editor.PreviousAutoEvoResults == null)
+        {
+            GD.PrintErr("Missing auto-evo results, can't show miches");
+            return;
+        }
+
+        var patch = Editor.CurrentPatch;
+
+        micheViewer.ShowMiches(patch, Editor.PreviousAutoEvoResults.GetMicheForPatch(patch),
```

Comment:
Hang on isn't this wrong? Miches are already viewable in the patch map tab. And the issue this is marked as closing calls specifically for showing miches for the *prediction* not the previous numbers. So this should show the data from `waitingForPrediction`.